openapi-spec/swagger.yaml

```diff
@@ -427,6 +427,37 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorSpec'
+  '/v1/storages/access-infos':
+    get:
+      tags:
+        - Storages
+      description: Get all access info of a registered storage backend
```

Comment:
description: Get  access info of all registered storages

Reply:
Modified